### DISCUSSION TITLE
Configure Keda without deploying to kube-system

### DISCUSSION THREAD

**Discussion by benedikt-mue** [kedacore/charts#321] [Q&A] [answered]
Hi,

Is it possible to setup keda as a non-privileged user in Kubernetes as I don´t have cluster-scope admin rights.
I am currently using the Helm chart and it seems that some configurations are not made configurable (eg keda-operator-auth-reader deployed to namespace `kube-system`). Thus, I wonder whether running keda in a dedicated namespace is actually possible?

Any suggestions or links that point me in the right direction are much appreciated. Cheers

**Comment by tomkerkhove** [comment, marked as answer]
You can install KEDA in any namespace that you want, however, it needs to set some cluster-wide settings for it to work such as the metric server, CRDs, etc.

**Reply by benedikt-mue** [reply]
thanks for the response.. that´s what I figured..
